inboundemailactivity.php

```diff
@@ -145,6 +145,27 @@ function inboundemailactivity_civicrm_themes(&$themes) {
 
 function inboundemailactivity_civicrm_pre($op, $objectName, $id, &$params) {
   if ($objectName === 'Activity' && $op === 'create' && $params['activity_type_id'] === CRM_Core_PseudoConstant::getKey('CRM_Activity_BAO_Activity', 'activity_type_id', 'Inbound Email')) {
+    // fetch the inbound email-to-activity email address contact, if no contact found then create one and use it for 'Added by' or source_contact_id 
+    // if not found ignore and let the source contact be the From contact
+    $emailToActivityContact = civicrm_api3('MailSettings', 'get', [
```

Comment:
@monishdeb how are you limiting is to the email to activity rather than bounce? also what if there are multiple email to activity processors?

Reply:
So I found that the is_default = 0 is meant for 'email to activity' mail setting, as per the core code [here](https://github.com/civicrm/civicrm-core/blob/master/CRM/Admin/Form/MailSettings.php#L88)

Reply:
right yep ok